app/src/Sidebar/Positions/index.js

```diff
@@ -90,27 +94,26 @@ const Positions = ({
         positions,
         positionBalances
       );
-      setPositionGroups(positionGroups);
+      setPositionGroups(
+        positionGroups.filter((positionGroup) => {
+          const { amount } = positionGroup;
+          const isPositionLargeEnough = amount.lt(1e12);
```

Comment:
Shouldn't here be Greater Than?

Reply:
No, but the name is wrong! Thank you, will fix